src/Nordea/Components/DatePicker.elm

```diff
@@ -0,0 +1,410 @@
+module Nordea.Components.DatePicker exposing (InternalState(..), OptionalConfig(..), init, view)
```

Comment:
Could we make the internal state opaque? That way, if we want to make changes to it, we only need to update this file. The only way to construct it is via functions in this file, for example the init function
```suggestion
module Nordea.Components.DatePicker exposing (InternalState, OptionalConfig(..), init, view)
```